packages/@okta/vuepress-theme-default/components/TableOfContents.vue

```diff
@@ -68,6 +69,12 @@ export default {
     }
   },
   methods: {
+    scrollToItem(hash) {
+      let target = document.querySelector(hash)
+      if(target) {
+        window.scrollTo(0, (target.offsetTop - 40))
```

Comment:
why 40?  Perhaps a comment to help the next poor soul?

Reply:
I plan to rebuild all of this after release.  40 is the needed distance for getting the header out from hiding behind the fixed nav.